org.opentosca.bus/org.opentosca.bus.application.api.jsonhttp/src/main/java/org/opentosca/bus/application/api/jsonhttp/servicehandler/ApplicationBusServiceHandler.java

```diff
@@ -1,38 +0,0 @@
-package org.opentosca.bus.application.api.jsonhttp.servicehandler;
-
-import javax.inject.Inject;
-import javax.inject.Singleton;
-
-import org.opentosca.bus.application.service.IApplicationBusService;
-import org.springframework.stereotype.Service;
-
-/**
- * Service Handler of the Application Bus-JSON/HTTP-API.<br>
- * <br>
- * <p>
- * Here the implementation of the IApplicationBusService is binded or unbinded. During the binding the routing endpoint
- * of the Application Bus is handed over.
- *
- * @author Michael Zimmermann - zimmerml@studi.informatik.uni-stuttgart.de

```

Comment:
Does removing this not kill the App Bus ?

Reply:
Legacy code from previous refactorings. It wasn't used anymore.